internal/applicationsnapshot/input_test.go

```diff
@@ -223,3 +302,42 @@ func TestReadSnapshotFile(t *testing.T) {
 	})
 
 }
+
+func TestExpandImageIndex(t *testing.T) {
```

Comment:
It would be great to cover some of the error statements.

Reply:
Done :)